evm-cpu.py

```diff
@@ -337,12 +340,17 @@ def is_arithmetic(self):
                 0x3a: ('GASPRICE', 0, 0, 1, 2, 'Get price of gas in current environment.'),
                 0x3b: ('EXTCODESIZE', 0, 1, 1, 20, "Get size of an account's code."),
                 0x3c: ('EXTCODECOPY', 0, 4, 0, 20, "Copy an account's code to memory."),
+                0x3d: ('RETURNDATASIZE', 0, 0, 1, 2, "Pushes the size of the return data buffer onto the stack."),
+                0x3e: ('RETURNDATACOPY', 0, 3, 0, 3, "	Copies data from the return data buffer to memory."),
+                0x3f: ('EXTCODEHASH', 0, 1, 1, 400, "Returns the keccak256 hash of a contract's code."),
```

Comment:
EXTCODEHASH was changed to 700 in the Istanbul hard fork

Reply:
Thanks for the reminder, I'm new to blockchain.
I also learned that many opcodes in this repo such as SLOAD and BALANCE are out of date.